lib/src/main/scala/code/chymyst/jc/Molecules.scala

```diff
@@ -288,51 +295,82 @@ class M[T](val name: String) extends (T => Unit) with NonblockingMolecule[T] {
   def unapply(arg: UnapplyArg): Option[T] = unapplyInternal(arg)
 }
 
-/** This trait contains implementations of most methods for the [[ReplyValue]] and [[EmptyReplyValue]] classes.
+/** Represent the different states of the reply process.
```

Comment:
Yes, I tried do make Scaladocs, it works but generates some invalid links. I will certainly publish scaladocs when the project is mature enough to be published on Maven.